### PR TITLE
home page slowness selection fix

### PR DESCRIPTION
**Pull request by jodygarnett**
@aaime here is small fix for selection issue observed when using setResponsePage

### CODE REVIEW

**Comment by aaime** [comment]
Looks like other commits snuk in

**Comment by aaime** [comment]
That said, just reading the diff on my phone, looks legit. I'll check it better when I'm back home

**Comment by jodygarnett** [comment]
> Looks like other commits snuk in

arrgg! I will try and fix

File: src/web/core/src/main/java/org/geoserver/web/GeoServerHomePage.java [hunk 1]
```diff
@@ -252,12 +252,8 @@ void selectHomePage(String workspaceName, String layerName) {
...
 
         PageParameters pageParams = new PageParameters();
-        if (workspaceSelection != null) {
-            pageParams.add("workspace", workspaceSelection, 0, INamedParameters.Type.QUERY_STRING);
-        }
-        if (layerSelection != null) {
-            pageParams.add("layer", layerSelection, 1, INamedParameters.Type.QUERY_STRING);
-        }
+        pageParams.add("workspace", workspaceSelection, 0, INamedParameters.Type.QUERY_STRING);
+        pageParams.add("layer", layerSelection, 1, INamedParameters.Type.QUERY_STRING);
```
Comment (jodygarnett) [inline src/web/core/src/main/java/org/geoserver/web/GeoServerHomePage.java:256]: I still would love to use ``INamedParameters.Type.PATH`` but could not figure out the mapping.

**Pull request merged**